jbmc/src/java_bytecode/java_class_loader.h

```diff
@@ -102,8 +102,7 @@ class java_class_loadert:public messaget
   /// Load jar archive or retrieve from cache if already loaded
   /// \param limit
   /// \param filename name of the file
-  jar_filet &jar_pool(
-    java_class_loader_limitt &limit, const std::string &filename);
+  jar_filet &jar_pool(const std::string &filename);
```

Comment:
Here and below: doxygen update required.

Reply:
Done!